vale/styles/jargonLint/Licenses.yml

```diff
@@ -0,0 +1,40 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  # Apache[ -]?2(?:\.0)?: Apache 2.0 # also a webserver, to be fixed
+  "[0O]BSD": 0BSD
```

Comment:
Is there a reason why this is between quotation while the others aren't.

Reply:
[] defines an array

so here we are writing

[0O] (an array containing 0O) and concat "BSD"

yaml is broken

```
vale/styles/jargonLint/Licenses.yml
  10:7      error    syntax error: expected <block end>, but found '<scalar>' (syntax)
```

Reply:
This is the reason I added yamllint to CI

- #43

Reply:
I see, thanks for the explaination.

Reply:
E201 Invalid value [~/.local/share/vale/styles/jargonLint/Licenses.yml:9:1]:

   8  swap:
   9*   # Apache[ -]?2(?:\.0)?: Apache 2.0 # also a webserver, to be fixed
  10    [0O]BSD: 0BSD
  11    BSD: BSD   # also an operating system, but it's OK. BSD versions are not like BSD3 but 3BSD

did not find expected key

Here is what vale reports.

I also plan to validate the rules against vale directly, but yamllint is a no-brainer